ls8/cpu.c

```diff
@@ -37,14 +38,25 @@ void cpu_ram_write(struct cpu *cpu, int address, int value) {
 /**
  * ALU
  */
-void alu(struct cpu *cpu, enum alu_op op, unsigned char regA, unsigned char regB)
+void alu(struct cpu *cpu, unsigned char op, unsigned char regA, unsigned char regB)
 {
   switch (op) {
-    case ALU_MUL:
-      // TODO
-      break;
+    case ADD:
+      cpu->registers

```

Comment:
per the spec, we need to store the sum of the values in the two registers in regA